src/form-builder/components/FormList.jsx

```diff
@@ -23,6 +23,11 @@ export default class FormList extends Component {
     const data = this._getUpdatedFormList(this.props.data);
     return map(data, (rowItem, index) => (
       <tr key={rowItem.id}>
+        <td className="form-list-check-box">
+          { rowItem.published ? <input onChange={() => this.props.handleSelectedForm(rowItem) }
+            type="checkbox"
```

Comment:
we can just write it like "rowItem.published && <input onChange={() => this.props.handleSelectedForm(rowItem) }
            type="checkbox"'/>", ternary operator not required

Reply:
yes changed